lib/resty/core/uri.lua

```diff
@@ -38,10 +38,10 @@ if subsystem == "http" then
 elseif subsystem == "stream" then
     ffi.cdef[[
     size_t ngx_stream_lua_ffi_uri_escaped_length(const unsigned char *src,
-                                                 size_t len);
+                                                 size_t len, int type);
 
     void ngx_stream_lua_ffi_escape_uri(const unsigned char *src, size_t len,
-                                       unsigned char *dst);
+                                       unsigned char *dst, int type);
```

Comment:
We need PRs for stream lua module otherwise this change will break the stream lua module.

Reply:
#303
openresty/lua-nginx-module#1709
openresty/meta-lua-nginx-module#77